crates/cxx-qt-build/src/lib.rs

```diff
@@ -437,6 +461,11 @@ impl CxxQtBuilder {
     /// It is therefore best practice to specify features on your crate that allow downstream users
     /// to disable any qt modules that are optional.
     pub fn qt_module(mut self, module: &str) -> Self {
+        // Ensure that CMake and Cargo build.rs are not out of sync
+        if qt_modules_import().is_some() && !self.qt_modules.contains(module) {
+            panic!("When building with CMake, all Qt modules must be specified in QT_MODULES of cxx_qt_import_crate");
```

Comment:
```suggestion
            panic!("Qt module mismatch between cxx-qt-build and CMake!\n\
                        Qt module '{module}' was not specified in CMake!\n\
                        When building with CMake, all Qt modules must be specified with the QT_MODULES argument in cxx_qt_import_crate");
```
I always find it worthwhile to make these errors as "foolproof" as possible :)
At the most basic level it will hopefully help understand our own design decisions a few months from now :sweat_smile:

Reply:
done